helpers/GenerateResourcesAndImage.ps1

```diff
@@ -263,6 +269,14 @@ Function GenerateResourcesAndImage {
         Write-Host "Restricting access to packer generated VM to agent IP Address: $AgentIp"
     }
 
+    if ($tags) {
+        $builderScriptPath_temp = $builderScriptPath.Replace(".json", "-temp.json")
+        $packer_script = Get-Content -Path $builderScriptPath | ConvertFrom-Json
+        $packer_script.builders | Add-Member -Name "azure_tags" -Value $ARM_TAGS -MemberType NoteProperty
```

Comment:
What is $ARM_TAGS variable?

Reply:
> What is $ARM_TAGS variable?

:o great see, it causing by a sandbox test environment otherwhere, and I forgot to replace this part. Thanks for your sphinx eye ;)
New Commit for fix it -> https://github.com/actions/virtual-environments/pull/5467/commits/9d7f701933df83278522aeb71caed882e5e0b610